test/e2e/network_test.go

```diff
@@ -25,6 +25,42 @@ func removeConf(confPath string) {
 	}
 }
 
+// generateNetworkConfig generates a cni config with a random name
+// it returns the network name and the filepath
+func generateNetworkConfig(p *PodmanTestIntegration) (string, string) {
+	// generate a random name to preven conflicts with other tests
+	name := "net" + stringid.GenerateNonCryptoID()
+	path := filepath.Join(p.CNIConfigDir, fmt.Sprintf("%s.conflist", name))
+	conf := fmt.Sprintf(`{
+		"cniVersion": "0.3.0",
+		"name": "%s",
+		"plugins": [
+		  {
+			"type": "bridge",
+			"bridge": "cni1",
+			"isGateway": true,
+			"ipMasq": true,
+			"ipam": {
+				"type": "host-local",
+				"subnet": "10.99.0.0/16",
```

Comment:
Thank you for your patience with me. I'm not sure this is going to work, because podman doesn't just trigger on name conflicts, also on subnet conflicts. I don't believe podman (or CNI?) will allow two networks with the same IP range. 

I wonder if you could use `"10.%d.0.0/16", 99 + GinkgoParallelNode()`? This is going to need weigh-in from networking and ginkgo SMEs, I fear.

Reply:
The tests never run this network config. It is only used for network ls/inspect/rm.
That's why we shouldn't run into subnet conflicts. But generell speaking it would definitely  better to generate random subnets to prevent misuse.

Reply:
Okay... I've convinced myself (via reading) that none of these tests (currently) do anything other than `ls` and `rm`; and furthermore that no other tests in `test/e2e` (currently) use the `10.99/16` subnet; and, finally (via manual testing) that `podman network create` does not (currently) do post-hoc testing of existing `.conflist` files and throw an error if two existing files conflict. Whew! So in short, yes, I believe this is likely to address some of the network flakes we're seeing, without introducing new problems. But would you consider documenting this potentially fragile set of assumptions? Some future day the tests will break again, and someone might notice the same-subnet question; a comment may save them some time. Thank you.